packages/mdc-textfield/README.md

```diff
@@ -236,6 +276,7 @@ text, validation, and dense UI.
 <label for="css-only-textfield-box">Your name:</label>
 <div class="mdc-textfield mdc-textfield--box">
   <input type="text" class="mdc-textfield__input" id="css-only-textfield-box" placeholder="Name">
+  <div class="mdc-textfield__bottom-line"></div>
```

Comment:
Is this actually needed for CSS-only textfield box? It's not included in our demo example.

Reply:
unnecessary. removed